src/figures.ts

```diff
@@ -1170,6 +1170,21 @@ export class Draw extends Frame {
   public sendRubberBandsMultiplot(figures: Figure[]): void {}
 
   protected receiveRubberBandFromFigure(figure: Figure): void {}
+
+  public highlightFromReferencePath(highlightData) {
+    const highlight = highlightData.highlight;
```

Comment:
I think we need to type highlightData here comming from new file Interactions.ts